Introduction to Coroutines and Channels/06_StructuredConcurrency.md

```diff
@@ -1,12 +1,12 @@
 # Structured concurrency
  
 _Coroutine scope_ is responsible for the structure and parent-child relationships between different coroutines.
-You always start new coroutines inside a scope.
+New coroutines always start inside a scope.
```

Comment:
I want to emphasize here that new coroutines are started not by themselves, but we start them. Therefore it's either 'new coroutines are always started' or 'we always start...'